person.rb

```diff
@@ -13,9 +9,15 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] nice work. please ensure that Getters and Setters for `@rentals` should be present

Reply:
I did that thanks so much! 😄